### DISCUSSION TITLE
Minor Improvement Ideas

### DISCUSSION THREAD

**Discussion by JeanTekk** [tamland/airsonic-refix#46] [Ideas]
I want to start with a big thanks to everyone contributing in this project!
It is awsome to have a way more intitive ui for arsonic than the shipped one with the server.
With docker and traefik it is also extremly easy to setup.

After using airsonic reflix for quite a while i had some ideas to make it even more better:
1. Make the container for the side menu sticky or fixed:
If i scroll through my collection or a playlist and want to drag and drop the item to a playlist, I have to scroll up again and that can take quite a while. (Maybe also with the attribute overflow: scroll)
2. I would love to see a PlayButton dicrectly on the covers. For example to be able to start an album directly from the albums view.
3.  The progress bar could be a little bit thicker or a thicker transparent wrapper bar could be placed behind it that also catches mouseclicks.

Thanks for your work!

**Comment by tamland** [comment]
Thanks for the feedback!  
I have replaced the progress bar with a different one. Should be a lot easier to use now.
